README.md

```diff
@@ -86,6 +86,7 @@ A brief explanation of what the config parameters mean:
  * `boot_command` - a list of commands to be sent to the instance over VNC.
  * `vm_other_config` - a map of string to string for setting vm's other-config.
  * `network_names` - a list of network name for the VIFs to connect with.
+ * `compressed_export` - use compression export or not.
```

Comment:
@makunterry - rather than adding an extra option, given this is only relevant to xva export, can we please add it as an export type?

Say `xva_compressed`.

This means that people are not confused into thinking you can use the option with say a vhd.